metricbeat/module/kubernetes/pod/data.go

```diff
@@ -40,7 +40,7 @@ func eventMapping(content []byte, perfMetrics *util.PerfMetricsCache) ([]common.
 	nodeCores := perfMetrics.NodeCoresAllocatable.Get(node.NodeName)
 	nodeMem := perfMetrics.NodeMemAllocatable.Get(node.NodeName)
 	for _, pod := range summary.Pods {
-		var usageNanoCores, usageMem, availMem, rss, workingSet, pageFaults, majorPageFaults int64
+		var usageNanoCores, usageMem, availMem, rss, workingSet, pageFaults, majorPageFaults uint64
```

Comment:
We are currently defining these values as `long` in `fields.yml`. We probably need to change the mapping to be able to store big numbers. `unsigned_long` is only supported since 7.10, maybe we can use `double` for this.

Reply:
I see, thanks for catching this! But does this mean that we can only ship it a new minor release right? Otherwise, with patch releases it might cause mapping conflicts?

Reply:
Yes, let's backport this only till 7.11.